src/plugins/inputs/join.js

```diff
@@ -0,0 +1,33 @@
+"use strict";
+
+var Chan = require("../../models/chan");
+var _ = require("lodash");
+
+exports.commands = ["join"];
+
+exports.input = function(network, chan, cmd, args) {
+	var client = this;
+	if (args.length !== 0) {
+		var irc = network.irc;
+		irc.join(args[0], args[1]);
```

Comment:
I was not thinking in terms of saving code but more for robustness. Here we need to parse things properly, send them back properly, while the command coming from the client should be a valid command, right?
If so, why not forwarding it as-is to `irc-framework` and deal with logic specific to us only in this handler?